### PR TITLE
ENH 11 rc1

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
cc @isuruf - We may have issues here with linking back to the apple libcxxabi on OSX 11 - we'll have to check

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
 package:
   name: libcxx_and_libcxxabi
   version: {{ version }}
 
 source:
-  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version }}/libcxx-{{ version }}.src.tar.xz
-    sha256: def674535f22f83131353b3c382ccebfef4ba6a35c488bdb76f10b68b25be86c
+  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/libcxx-{{ version.replace(".rc", "rc") }}.src.tar.xz
+    sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```
Comment (beckermr) [inline recipe/meta.yaml:9]:
```suggestion
    sha256: 7718cb9e60d5c7b7cf6e6fff1392397d92520f6c836d7185554b275b44b5726d
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     patches:
       # See https://bugs.llvm.org/show_bug.cgi?id=39696
       - ppc64le-constexpr.patch  # [ppc64le]
 
-  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version }}/libcxxabi-{{ version }}.src.tar.xz  # [linux]
-    sha256: a97ef810b2e9fb70e8f7e317b74e646ed4944f488b02ac5ddd9c99e385381a7b  # [linux]
+  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/libcxxabi-{{ version.replace(".rc", "rc") }}.src.tar.xz  # [linux]
+    sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  # [linux]
```
Comment (beckermr) [inline recipe/meta.yaml:15]:
```suggestion
    sha256: 944b6f9a46f7b1d119be9995534bedc2398602dd75ffa15c948c4e87e1031be8  # [linux]
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
Merged to rc branch

File: conda-forge.yml [hunk 3]
```diff
@@ -2,3 +2,4 @@ provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure, osx: travis}
 travis:
   osx_image: xcode6.4
 conda_forge_output_validation: true
+channel_priority: flexible
```
Comment (beckermr) [inline conda-forge.yml:5]: Why was this needed?

Comment (isuruf) [reply conda-forge.yml:5]: `conda-forge/label/llvm_rc` had `conda-forge` were conflicting

Comment (beckermr) [reply conda-forge.yml:5]: ohhh weird - did you understand the source of the conflict?

**Comment by beckermr** [comment]
@isuruf Do we need to update smithy to support the travis edits here?

**Comment by isuruf** [comment]
Yes, that would be nice to have.
